tests/test_wo_with_no_signature.py

```diff
@@ -0,0 +1,256 @@
+#! /usr/bin/env python3
+
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import random
+import json
+import argparse
+import logging
+import base64
+import secrets
+
+import config.config as pconfig
+import utility.logger as plogger
+from service_client.generic import GenericServiceClient
+import utility.utility as utility
+from utility.tcf_types import WorkerType
+import worker.worker_details as worker
+from work_order.work_order_params import WorkOrderParams
+from connectors.direct.direct_json_rpc_api_connector \
+	import DirectJsonRpcApiConnector
+import crypto.crypto as crypto
+from error_code.error_status import WorkOrderStatus
+
+# Remove duplicate loggers
+for handler in logging.root.handlers[:]:
+    logging.root.removeHandler(handler)
+logger = logging.getLogger(__name__)
+TCFHOME = os.environ.get("TCF_HOME", "../../")
+
+def ParseCommandLine(args) :
+
+	global worker_obj
+	global worker_id
+	global message
+	global config
+	global off_chain
+
+	parser = argparse.ArgumentParser()
+	use_service = parser.add_mutually_exclusive_group()
+	parser.add_argument("-c", "--config",
+		help="the config file containing the Ethereum contract information",
+		type=str)
+	use_service.add_argument("-r", "--registry-list",
+		help="the Ethereum address of the registry list",
+		type=str)
+	use_service.add_argument("-s", "--service-uri",
+		help="skip URI lookup and send to specified URI",
+		type=str)
+	use_service.add_argument("-o", "--off-chain",
+		help="skip URI lookup and use the registry in the config file",
+		action="store_true")
+	parser.add_argument("-w", "--worker-id",
+		help="skip worker lookup and retrieve specified worker",
+		type=str)
+	parser.add_argument("-m", "--message",
+		help='text message to be included in the JSON request payload',
+		type=str)
```

Comment:
Add -s command line argument to existing echo client implementation. If -s passed then  add requesterSignature otherwise no signature. No need to add entire python code as test case, call "echo_client.py -s" option in run_tests.sh script.

Reply:
I have added -r option to command line argument to enable requester signature. 
Also, i will remove my test cases